parsing/lexer.mll

```diff
@@ -413,7 +413,11 @@ rule token = parse
         let s, loc = with_comment_buffer comment lexbuf in
         COMMENT (s, loc) }
   | "(*" ('*'*) as stars "*)"
-      { COMMENT (stars, Location.curr lexbuf) }
+      { if !handle_docstrings && stars="(*" then
```

Comment:
Is this right? I thought stars should equal "" here since it should only be bound to the `('*'*)`. If that is not the case then I think there are other bugs in the lexer's handling of stars elsewhere.